asserts/ifacedecls.go

```diff
@@ -235,10 +235,17 @@ func (c *AttributeConstraints) Check(attrs map[string]interface{}) error {
 	return c.matcher.match("", attrs)
 }
 
+// OnClassicConstraint specifies a constraint based whether the system is classic and optional specific distros' sets.
+type OnClassicConstraint struct {
+	Classic bool
+	Distros []string
```

Comment:
Is 'Distros' the right word here? I mean, we all know what it is, but I wonder if distribution is better and less informal (here and in error messages). If this term went through review with Gustavo and/or snappy designers already, then don't change anything. Otherwise, maybe just run it by Gustavo real quick to see if it should change.